### ISSUE TITLE
Installing non-standard kernel with nvidia proprietary driver won't boot into GFX mode

### ISSUE THREAD

**Issue by mkzero** [deadhead420/anarchy-linux#656]
When choosing a non-standard kernel(in my case linux-zen) from the installer menu and then choosing the proprietary nvidia driver will fail to boot. 

Installing the DKMS nvidia driver instead solved the problem for me(not 100% happy with that since I have had one or two incidents that had me switch from nvidia-dkms to nouveau because of some incompatibilities with the new kernel), so maybe for those kernels if there's not a package in the Arch repos and noone wants to build the AUR package into the Anarchy repos the best option would be to install the DKMS driver?

**Comment by JustMeErazem** [comment]
@izzy Hi, can you please retry the installation with 1.0.4 and if you the issue re-occurs, send some logs, if possible.

**Comment by ShayBox** [comment]
Yes, this still happens, you need to install nvidia-dkms instead of nvidia if any kernel other than `linux` was chosen.

**Comment by ShayBox** [comment]
It's the preferred way because dkms modules work with any kernel as long as you install the headers, they don't need to be updated with the kernel or kept in sync with it, I use dkms even with the stock kernel, most projects don't even make a non dkms module anymore, so it's already installed for me (responding to the first message from 19 hours ago)

**Comment by ShayBox** [comment]
Basically you install linux-headers (or your custom kernel-headers) and nvidia-dkms instead of Nvidia, will just work, if you didn't know how to get it working

**Comment by thomaslawton91** [comment]
Wanted to bump this rather than opening a new issue. I did a clean install using the zen kernel and nvidia proprietary graphics and had this issue. I could _not_ figure out what was wrong until I realized that the graphics that were being installed were not dkms. I'm happy to use dkms but it was a difficult problem to troubleshoot considering.
